src/main.rs

```diff
@@ -77,96 +110,252 @@ pub async fn main() {
 
     let args = CliArgs::parse();
 
-    bootstrap(&args).await;
+    println!("{color_red}{}{color_reset}", BANNER);
+
+    let config = Config::init(&args).await;
+
+    let output = match args.mode {
+        BootstrapMode::Full => bootstrap(&args, &config).await,
```

Comment:
should we just remove the `Full` command, I don't think there's any scenario where this can be used right? the same for SetupL2 now that I think of it. you need to restart madara with different fees? maybe `SetupL2` can be 

- we pause where needed with a message saying restart madara XYZ
- OR, once eth bridge is setup, it will set all other things with fees

Reply:
yes I don't think there can be full as per the current implementation, we anyway have to restart madara with the core contract address